CHANGELOG.md

```diff
@@ -17,6 +18,89 @@ The below sections record the changes for each API version in each release as fo
 * for subsequent alpha or release-candidate API versions, the delta with respect to the previous pre-release
 * for a public API version, the consolidated changes since the release of the previous public API version
 
+# r1.3
+## Release Notes
+
+This patch release contains the definition and documentation of
+* device-roaming-status v0.6.0
+* device-roaming-status-subscriptions v0.6.1
+* device-reachability-status v0.6.1
+* device-reachability-status-subscriptions v0.6.1
+
+The API definition(s) are based on
+* Commonalities v0.4.0
+* Identity and Consent Management v0.2.0
```

Comment:
you could update the ICM version to 0.2.1 (there is no API impact)

Reply:
version changed